n_t/Snakefile

```diff
@@ -155,39 +152,58 @@ rule homo_sapiens_compound_smcpp:
         plots.plot_compound_smcpp(input, output[0])
 
 
+# ###############################################################################
+# MSMC CONSTRUCTION ZONE
+# ###############################################################################
+
+
+rule homo_sapiens_ts_to_multihep:
+    input: rules.homo_sapiens_Gutenkunst_simulation.output
+    output: "homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees.multihep.txt"
+    run: msmc.write_msmc_file(input[0])
+
+
 rule homo_sapiens_msmc:
     input:
-        "homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees"
-    output:
-        "homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees.multihep.txt.final.txt"
+        #TODO make this not dependent on all simulations. 
+        # seed = output.seed?
+        expand("homo_sapiens_Gutenkunst/{seeds}.{chrms}.trees.multihep.txt", \
```

Comment:
The line breaking is a bit inconsistent here. Do we really need the backslash? (If it was Python code we wouldn't). The second line should definitely be indented though, it's very confusing having a broken line at the same indentation level.